### PR TITLE
update to script 2_attributePoint.R with stars

### PR DESCRIPTION
**Pull request by ptmcintyre**
first attempt to use stars package to address better raster processing for large ranges/many points.
Could not combine all rasters into a single stars object, so currently reads each raster in as a stars object and uses st_extract within a loop.
no parallelization used but seems quite fast without
need to understand why some files cannot be combined in stars (e.g. clim_bio1 and beersx1000 -different crs?)

### CODE REVIEW

**Comment by tghoward** [comment]
Thanks for updating.  In my first test I'm getting this error:
```
Error in match.names(clabs, names(xi)):
    names do not match previous names
```
I'll go a little deeper.

File: 2_attributePoints.R [hunk 1]
```diff
@@ -119,49 +120,77 @@ fullL <- gridlist[names(gridlist) %in% tolower(gridlistSub$gridName)]
...
+  e.df_list[[i]]<-as.data.frame(e.df_extract)   #
 }
 # stop cluster
-sfStop()
+#sfStop()
 
-e.df<-do.call(rbind, e.df_list)
 
+
+e.df<-do.call(rbind, e.df_list)
```
Comment (tghoward) [inline 2_attributePoints.R:188]: I need to use `cbind` here, in my test

Comment (tghoward) [reply 2_attributePoints.R:188]: can you confirm?

**Comment by tghoward** [comment]
Aside from the `cbind` comment, there are other minor things, such as  a better way to clear the geometry column. Is there a "Allow edits from maintainers' checkbox somewhere, that you can check? Then I could made some of these minor tweaks.

File: 2_attributePoints.R (hunk 1 above)
Comment (ptmcintyre) [reply 2_attributePoints.R:188]: yes- Thought I'd fixed that but must not have pushed that particular change
should be cbind

**Comment by ptmcintyre** [comment]
Allow edits from maintainers is checked. Perhaps I need to add you with permissions to my fork. I'll do that.

**Review by tghoward — APPROVED** [review]

**Comment by tghoward** [comment]
It runs!  We can make clean-up tweaks after pulling

**Pull request merged**